python_modules/dagster/dagster/utils/timing.py

```diff
@@ -1,12 +1,22 @@
-from contextlib import contextmanager
+import os
+import sys
 import time
 
+from contextlib import contextmanager
+
 from dagster import check
 
+if sys.version_info.major >= 3 and sys.version_info.minor >= 3:
```

Comment:
if even have a few more of these we should actually create the seven library

Reply:
just to flex?